src/main/resources/liquibase/workbench_changelog/16_1_0.xml

```diff
@@ -15,5 +15,559 @@
 			DROP TABLE workbench_sidebar_category_link_role;
 		</sql>
 	</changeSet>
+	<changeSet author="diego" id="v16.1.0-2">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1"> select count(rp.role_id) > 0 from role_permission rp inner join permission p on rp.permission_id = p.permission_id where p.name = 'BREEDING_ACTIVITIES'</sqlCheck>
+        </preConditions>
+        <comment>Remove Breeding activities permission from role_permission table</comment>
```

Comment:
Remove breeding activities permission assignment for role_permission table and giving permissions to its children